WebContent/HomeWorkPage.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>HomeWork Page</title>
+</head>
+<body style="background-color:black;color:white;">
+  <h1>HomeWork Page</h1>
+  <hr>
+  <img alt="" src="berlin.jpg">
+  <ul type="square">
+  <li><a>https://github.com/x3matt</a></li>
```

Comment:
This should be a link the user can click in the browser. Check the doc what is 'href' attribute of 'a' tag